src/scheduler.cc

```diff
@@ -13,13 +13,25 @@
 namespace celerity {
 namespace detail {
 
-	abstract_scheduler::abstract_scheduler(const size_t num_nodes, const node_id local_node_id, const system_info& system, const task_manager& tm,
-	    delegate* const delegate, command_recorder* const crec, instruction_recorder* const irec, const policy_set& policy)
+	const char* print_command_type(const command& cmd) {
+		return matchbox::match(
+		    cmd,                                                    //
+		    [](const epoch_command&) { return "epoch"; },           //
+		    [](const horizon_command&) { return "horizon"; },       //
+		    [](const execution_command&) { return "execution"; },   //
+		    [](const push_command&) { return "push"; },             //
+		    [](const await_push_command&) { return "await push"; }, //
+		    [](const reduction_command&) { return "reduction"; },   //
+		    [](const fence_command&) { return "fence"; });
+	}
+
```

Comment:
This being in `scheduler.cc` seems a bit off.
Obviously it's currently only used here, but I'd expect to find it in print utils or so.

Reply:
Moved to `print_utils.h`.